metadata.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "CLDF dataset of the supplementary materials to "Subgrouping in a ‘dialect continuum’: A Bayesian phylogenetic analysis of the Mixtecan language family" (Auderset et al. 2023. JoLE)",
+    "title": "CLDF dataset of the supplementary materials to \"Subgrouping in a ‘dialect continuum’: A Bayesian phylogenetic analysis of the Mixtecan language family\" (Auderset et al. 2023. JoLE)",
```

Comment:
We use short titles here, please change to 

```
CLDF dataset accompanying Auderset et al.'s `"Subgrouping in a Dialect Continuum\" from 2023
```

This is our standard formula for papers of this form.

Reply:
ok, got it.